internal/logging/zerolog.go

```diff
@@ -81,6 +83,20 @@ func cloudwatchWriter() (io.Writer, func(), error) {
 	return cloudWatchWriter, cloudWatchWriter.Close, nil
 }
 
+// sentryWriter creates a zerolog writer for sentry.
+// Uses github.com/archdx/zerolog-sentry which is very simple wrapper.
+func sentryWriter(dsn string) (io.Writer, func(), error) {
+	wr, err := zlogsentry.New(dsn)
+	if err != nil {
+		return nil, func() {}, fmt.Errorf("cannot initialize sentry: %w", err)
+	}
+	sentry.ConfigureScope(func(scope *sentry.Scope) {
+		scope.SetTag("stream", config.BinaryName())
+	})
```

Comment:
This doesn't seem to work :/

Reply:
I've needed to copy the writer in our project to use `sentry.Init`
I've sent https://github.com/archdx/zerolog-sentry/pull/5 once that's in, we can switch to using that package.